var/spack/repos/builtin/packages/munge/package.py

```diff
@@ -43,6 +43,7 @@ def configure_args(self):
         localstatedir = self.spec.variants["localstatedir"].value
         if localstatedir != "PREFIX/var":
             args.append("--localstatedir={0}".format(localstatedir))
+        args.append(f"--with-pkgconfigdir={self.spec.prefix}/lib/pkgconfig")
```

Comment:
Also this one can possibly be a fix on `munge`

Reply:
Oh, I forgot about this one, yes this can split out too.  Crazy bug, it only installs a pkgconfig file if you set the target explicitly or there is a /usr/lib/pkgconf directory. 🤷